cinema/views.py

```diff
@@ -46,7 +52,7 @@
 ):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The permission class `IsAdminOrIfAuthenticatedReadOnly` is correctly applied to the `ActorViewSet`, ensuring that only admins can create actors while authenticated users can read them.